test/test_mypy.py

```diff
@@ -58,8 +61,9 @@ class TestPymongo(unittest.TestCase):
     coll: Collection
 
     @classmethod
+    @client_context.require_connection
     def setUpClass(cls) -> None:
-        cls.client = MongoClient(serverSelectionTimeoutMS=250, directConnection=False)
+        cls.client = rs_or_single_client(serverSelectionTimeoutMS=250, directConnection=False)
```

Comment:
Let's make this class inherit from IntegrationTest, then we get a `self.client` and `client_context.require_connection` for free.

Reply:
Done